### ISSUE TITLE
Can I update to 14.6 ??

### ISSUE THREAD

**Issue by lestitaniseur** [assafdori/bypass-mdm#6]
Hey I used this script to bypass mdm in macOS Sonoma 14.5 can I update to 14.6 ??
Also is macOS 15 public beta supported ??

**Comment by assafdori** [comment]
It works fine on 14.6 and _should_ work on MacOS 15, let me know if it doesn't.

**Comment by lestitaniseur** [comment]
> It works fine on 14.6 and _should_ work on MacOS 15, let me know if it doesn't.

yeah but you tested ??
but if I have the mdm message in the updating what should I do ans please can you test for me thank a lot

**Comment by assafdori** [comment]
I didn't test on MacOS 15 yet and I won't in the near future. I suggest you try and let me know if it does not. I can assist in troubleshooting. As far as I know they haven't changed how MDM gets provisioned so it should work just fine.

**Comment by lestitaniseur** [comment]
yeah ok imagine am getting the mdm msg In the config of macOS 14.5/15 then I do the steps for the mdm bypass and didn't work then how I back to the 14.5 ??

**Comment by assafdori** [comment]
What errors did you get? It's most likely not related to the OS version.

**Comment by lestitaniseur** [comment]
I didn't  do the update yet but I was afraid to get the mdm msg 



> What errors did you get? It's most likely not related to the OS version.

**Comment by assafdori** [comment]
If you've already disabled the MDM on 14.5 for example, it should not re-prompt you to configure the MDM on 15.

**Comment by Alqibla** [comment]
14.5 work fine ... 14.6 not test

**Comment by lestitaniseur** [comment]
i tested 14.6.1 work well anyone tested in 15 ???

**Comment by KITGLOBAL** [comment]
Guys i have updated to 15.0 Beta (24A5331b) and all is fine.

**Comment by sameastham** [comment]
How are y'all getting it to work in 14.6.1?? It makes me go through all the MDM process first. If I don't do that, then when I go back into recovery it says that there's no user to recover. So then I have to go back and go through MDM registration, at which point it starts to download all the policies.

**Comment by mateussiqueira** [comment]
You can safely update from 14.x to any newer version. The bypass survives the update in most cases. If MDM reappears, just re-run the script.

The v3 script with daemon disable survives updates much better than the legacy v1, which only touches temporary markers.
